Models/CharityEvent.cs

```diff
@@ -15,13 +15,16 @@ public class CharityEvent : ICharityEvent
 
         public List<Volunteer> Volunteers { get; set; }
 
-        public CharityEvent(string name, string description, Guid id, CharityEventState state = CharityEventState.Undefined)
+        public string ManagedByUser { get; set; }
+
+        public CharityEvent(string name, string description, Guid id, string managedByUser, CharityEventState state = CharityEventState.Undefined)
```

Comment:
would change managedByUser to managerId and use Identity ID

then we wouldn't risk problems with same usernames